core/src/main/scala/neotypes/DeferredQuery.scala

```diff
@@ -367,44 +378,64 @@ final class DeferredQueryBuilder private[neotypes] (private val parts: List[Defe
     */
   def query[T]: DeferredQuery[T] = {
     @annotation.tailrec
-    def loop(remaining: List[Part], queryBuilder: StringBuilder, accParams: Map[String, QueryParam], nextParamIdx: Int): DeferredQuery[T] =
+    def loop(remaining: List[Part],
+             queryBuilder: StringBuilder,
```

Comment:
I think we should move the `queryBuilder` argument outside of the `loop`, it is mutable in any case.

Reply:
Sure, I can do that. It's a bit weird right now. Feels like a leftover from something else